python/cudf/cudf/core/udf/utils.py

```diff
@@ -254,8 +263,14 @@ def _get_input_args_from_frame(fr):
 
 
 def _return_arr_from_dtype(dt, size):
+    extensionty = masked_array_types.get(dt)
+    if extensionty:
```

Comment:
Good place for a :seal: (no walrus emoji :cry:):
```suggestion
    if extensionty := masked_array_types.get(dt):
```